### DISCUSSION TITLE
Kafka not spinning up

### DISCUSSION THREAD

**Discussion by raghulp18** [strimzi/strimzi-kafka-operator#10212] [General]
Hi Strimzi Community,

I have deployed strimzi-cluster-operator in my local and got this config file from strimzi quickstart website. And I've also got the kafka persistence single yaml from the same site and I've deployed it in my local. But the kafka and entity operator is not spinning up, sometimes it's running, sometimes it's not running (waited for more than 1 hour). Only the zookeper is always spinning up.

![image](https://github.com/strimzi/strimzi-kafka-operator/assets/94948561/d586ccef-d3b5-445f-a7cd-418424b51bac)

Here is the kafka yaml as well 

<img width="720" alt="image" src="https://github.com/strimzi/strimzi-kafka-operator/assets/94948561/dabad779-49bb-498f-9e96-82bdb3d216a8">

Please find the below attached the strimzi cluster log in the compressed file

[cluster_log.txt.zip](https://github.com/user-attachments/files/15757530/cluster_log.txt.zip)

**Comment by im-konge** [comment]
The entity operator is being deployed after all ZK and Kafka Pods are up and running.
You mentioned that you used the quickstart example from the website, however that was moved to KRaft like cluster IIRC.
There is a ZK issue with forming the quorum, so I would scale the ZK replicas to 3 (which has less probability of hitting such issue) or tried the KRaft cluster.

Also, you should provide more logs than just log from CO -> logs from ZK, YAMLs of the CRs, etc.

**Comment by raghulp18** [comment]
Thanks for the quick respose. Please find the attached .zip file contains zookeeper logs and yamls.

[Archive.zip](https://github.com/user-attachments/files/15759061/Archive.zip)

**Reply by im-konge** [reply]
Yeah as I mentioned, you can try to scale to 3 replicas for ZK, delete the ZK Pods (and let it be re-created), if it solves the issue.

**Reply by raghulp18** [reply]
Thanks @im-konge, it worked !!
